components/reddit/common.js

```diff
@@ -0,0 +1,32 @@
+const reddit = require("./reddit.app");
+const get = require("lodash.get");
+
+module.exports = {
+  dedupe: "unique",
```

Comment:
Yes, agree. All were dedupe so far, but anyway they were moved to each source file, just in case they change in the future the changes will be per source file.